### DISCUSSION TITLE
Server Error 500 - changes not applied

### DISCUSSION THREAD

**Discussion by Gstretch22** [opengisch/QField#2965] [Q&A]
I collected some data yesterday afternoon in qfield ( photo locations) which were pushed the cloud and have synced which is visible in qgis desktop. I went out later in the afternoon to complete some further work and again pushed the data to the cloud. 

I can see the data in changes- but when I ask Qfield to 'apply' it comes up with an error message 'server error 500'.... Is there another way I can download this data and add it to my project ? Or perhaps a solution to the server error ?

Thanks,

**Comment by MelodyPremaillon** [comment]
I have the same problem, did you managed to solve it?
Thanks

**Comment by suricactus** [comment]
@Gstretch22 @MelodyPremaillon can you share any details of your projects so we can investigate the logs? Project and user name are enough.

**Reply by MelodyPremaillon** [reply]
my project is terrain_Heric_v2 and ly username premaillon.
Tank you!

**Reply by suricactus** [reply]
@MelodyPremaillon you can see two jobs triggered by me to test the fix for your problem. You should no longer see error 500 after trying to apply the deltas.

However, they are still not applied due to another error I am investigating.

**Comment by suricactus** [comment]
@MelodyPremaillon ok, I found it. You are using shapefiles as you data storage. As we noted in several places, the officially supported data storages are either GeoPackages or PostGIS layers. So you either change to one of these two data storages, or you should change your shapefiles to have a unique column names `fid`. Even in that case, note shapefiles are experimental.

**Comment by gabgarcies** [comment]
Hi,

I have the same issue and I work with Geopackage. In my case, I think the problem might be related to the fid and fid_1, which is autocreated by Qfieldcloud when uploading changes.

I created a post explaining my case a while ago: https://github.com/opengisch/qfieldcloud/issues/324#issue-1226565609

Thank you,

Gabriel Garcies

**Comment by JUAN1936** [comment]
THE SAME HAPPENS TO ME, BUT WHAT HAPPENS TO ME IS THAT IT DOESN'T LET ME IDENTIFY MYSELF ON THE PC OR ON THE MOBILE.
WON'T LET ME OR CHANGE THE PASSWORD

**Comment by fransua1590** [comment]
Me pasa lo mismo no me deje entra mi usuario no me deja cambiar la contraseña solo sale server Error (500)

**Comment by suricactus** [comment]
@gabgarcies @JUAN1936 @fransua1590 We are aware our services are not functioning properly due to external factors that are not under our control. Doing our best to restore the service as soon as possible.

Please follow the thread here https://github.com/opengisch/QField/discussions/3050

**Reply by gabgarcies** [reply]
Yeah, thank you!

Even though, this post was not primarely related to yestarday's problem. It was about Not applied changes.

**Comment by MelodyPremaillon** [comment]
Hello,

Thank you for your quick answer.

My username is premaillon and my main project project is terrain_Heric_v2.

But I have problemes with all the projects :

  *   "test" is only for testing how qfield cloud works,
  *   with "terrain-Heric", I cannot push changes from my cellphone to the cloud, there is 0 changes but on my phone I ha ve reccords.). It was the first time I was using QField cloud, maybe I did something wrong.

Once again, thank you

Wish you a good afternoon

Mélody
________________________________
De : Ivan Ivanov ***@***.***>
Envoyé : mercredi 22 juin 2022 11:44
À : opengisch/QField ***@***.***>
Cc : Mélody P ***@***.***>; Mention ***@***.***>
Objet : Re: [opengisch/QField] Server Error 500 - changes not applied (Discussion #2965)


@Gstretch22<https://github.com/Gstretch22> @MelodyPremaillon<https://github.com/MelodyPremaillon> can you share any details of your projects so we can investigate the logs? Project and user name is enough.
